Form-Controls/styles.css

```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css2?family=Questrial&family=Roboto:wght@100&display=swap");
+
+body {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  min-height: 100vh;
+  font-family: "Questrial", sans-serif;
+  text-transform: uppercase;
+  background: rgb(224, 224, 231);
+  background: linear-gradient(
+    90deg,
+    rgba(224, 224, 231, 1) 0%,
+    rgba(158, 157, 157, 0.9276304271708683) 52%,
+    rgba(66, 70, 70, 1) 100%
+  );
+  /* font-family: "Roboto", sans-serif; */
```

Comment:
```suggestion

```
Please don't commit commented out code.

https://syllabus.codeyourfuture.io/guides/code-style-guide#dont-leave-lots-of-commented-out-code